packages/core/test/Config.ts

```diff
@@ -0,0 +1,27 @@
+import * as  chai from 'chai';
+import * as sinon from 'sinon';
+import { ConfigService } from '../src/Config';
+
+/// <reference types="node" />
```

Comment:
Are these "reference" lines still a thing :( ? The old version I still use privately has these (or tsconfig.json, which is my preference) but I was hoping that they got rid of these in the future versions.

Reply:
I don't think this is necessary if `@types/node` is installed, not sure off the top of my head.

Reply:
Code worked but visual studio complained. 
This is only to improve VisualStudio experience.